Makefile

```diff
@@ -13,24 +13,51 @@ no_test: programs
 
 programs: lib
 	$(MAKE) -C programs
+ifdef ENABLE_PSA
+	$(MAKE) -C crypto/programs
+endif
 
 lib:
+ifdef ENABLE_PSA
+	$(MAKE) -C crypto/library
+endif
 	$(MAKE) -C library
 
 tests: lib
 	$(MAKE) -C tests
+ifdef ENABLE_PSA
+	$(MAKE) -C crypto/tests
```

Comment:
As suggested above, I think we can remove this.

Reply:
Removed